### DISCUSSION TITLE
No warning on script syntax error

### DISCUSSION THREAD

**Discussion by culdep-sh** [burkybang/SuiteAdvanced-Feedback#83] [General]
Hi, I'm trying out this extension.
when I save a script with a syntax error, net suite will not save the file a return with a error page, but with this extension a green box with saved comes up but the file doesn't get saved.

**Comment by burkybang** [comment]
Sorry about that! I'll get it fixed. Thank you for reporting the issue.

**Comment by burkybang** [comment]
Created the issue here: https://github.com/burkybang/SuiteAdvanced-Feedback/issues/84

**Reply by culdep-sh** [reply]
Thanks!

p.s.
Just a suggestion, it would be very helpful if the NetSuite error format would be more readable

**Reply by burkybang** [reply]
Great idea 😀 

![image](https://user-images.githubusercontent.com/927886/235263578-cc48010d-702b-4463-9116-8e796a4749c1.png)

**Comment by burkybang** [comment]
Fixed in version 26.7.2

**Reply by culdep-sh** [reply]
Thank You!
